### DISCUSSION TITLE
DDrawCompat

### DISCUSSION THREAD

**Discussion by ghost** [FaithBeam/Sims-1-Complete-Collection-Widescreen-Patcher#59] [General]
DDrawCompat had an update where they added settings and, like, the ability to enable anti-aliasing. But I don't know if it works yet. Time to update :)

**Comment by FaithBeam** [comment]
Hello! I just released [v3.2.2](https://github.com/FaithBeam/Sims-1-Complete-Collection-Widescreen-Patcher/releases/tag/3.2.2) which includes the updated DDrawCompat. I haven't been able to get anti-aliasing working but I don't know if that's because I'm configuring it wrong. Give it a shot!

**Comment by ghost** [comment]
I mentioned this and the developer sent a file with configurations that include anti-aliasing. Unfortunately, on some processors, sims disappear and they disappeared from me, but maybe everything will work for you. (https://github.com/narzoul/DDrawCompat/issues/160)

**Reply by AVissies** [reply]
If possible, please add the anti-aliasing, thank you
